release.sh

```diff
@@ -1,8 +1,33 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
@rdallman this script will do an auto-bump from version stored at `fdk/version.py`, so we can be sure that it's not stale.